src/ViewProfile.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import {withRouter, Link} from 'react-router-dom'
+
+class ViewProfile extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        userId: this.props?.location?.state?.id,
+        name: "",
+        age: "null",
```

Comment:
just saying the age is now set to a default value of 0 so that it is an integer, this should be reflected as well in the birthday PR